common/static/common/css/base.css

```diff
@@ -111,9 +107,9 @@ input[type="color"],
     font-size: 13px;
     font-weight: normal;
     line-height: 1;
-    -webkit-border-radius: 15px;
-    -moz-border-radius: 15px;
-    border-radius: 15px;
+    -webkit-border-radius: 3px;
+    -moz-border-radius: 3px;
```

Comment:
awesome, i was actually thinking how silly it is to keep all of these but never bothered to google if they were necessary -- i think i copied it from a library at some point and they just stayed around. Not that I've been testing but ideally the eyebrowse site is compatible across browsers since you don't need the extension to view.